packages/contracts-bedrock/layout-lock.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Can you document how this file is supposed to work?

Reply:
Added to README